examples/remote_store/remote_server.py

```diff
@@ -21,7 +21,7 @@
     print("In run with UUID: %s" % run.info.run_uuid)
     tracking_uri = get_tracking_uri()
     if tracking_uri.startswith("http://"):
-        store = RestStore(get_tracking_uri())
+        store = mlflow.tracking.get_service().store
```

Comment:
This doesn't look right. We probably shouldn't condition on `tracking_uri.startswith("http://")`, since `mlflow.tracking.get_service().store` will work in any case. Additionally, `.store` isn't really an interface we expose, so this example should use only what's exposed directly by `get_service()`, but that's a more significant refactoring so we can leave it to later.